lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/LocalSubmissionFromCS.java

```diff
@@ -34,6 +34,7 @@
 
 import java.io.File;
 import java.util.ArrayList;
+import java.util.List;
```

Comment:
Nothing changed here besides the move from `ArrayList` to `List` in a method body. Maybe remove the change from this PR?